ProjectManagmentSystem/src/Modules/AuthenticationModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,147 @@
-import React from 'react'
 
+import logo from '../../../../Modules/../assets/images/PMS 3.svg'
+import { TextField } from '@mui/material';
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 export default function ResetPass() {
+  let  navigate = useNavigate()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit =  async (data) => {
+    console.log(data)
+    try {
+      let response = await axios.post("https://upskilling-egypt.com:3003/api/v1/Users/Reset",data)
+          toast.success('password rest successfu')
+      console.log(response)
+      navigate('/login')
+    }
+       catch (error) {
+      toast.error(error.response.data.message)
+
+    }
+
+  }
   return (
-    <div>ResetPass</div>
+    <div className='auth-container'>
+      <div className='container-fluid'>
+        <div className="row d-flex vh-100 justify-content-center align-items-center">
+          <div className="col-md-6">
+            <div className="forgot text-center mb-4">
+              <img src={logo} alt="logo" className='w-25' />
+            </div>
+
+            <form action="#" onSubmit={handleSubmit(onSubmit)} className='form-auth vh-25' style={{ padding: "30px 30px" }}>
+              <span className='welcome-pms'>welcome to PMS</span>
+              <h1 className='auth-title'>Reset  Password</h1>
+              <div className='auth-standard-basic'>
+                <span className='e-mail'>E-mail</span> <br />
+                <TextField className='mb-4 w-75' id="standard-basic" label="Enter your E-mail" variant="standard"
+                  type="text"
+                  {...register("email", {
```

Comment:
Email, password and other validation will be used in multiple places. I consider making a file containing all needed validations and then use this validation in our application. This will make our code more concise. 

You can look at this example:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/InputValidator.ts